### ISSUE TITLE
Metadata - údaj o číslu

### ISSUE THREAD

**Issue by krizovj** [incad/K5Journals#68]
Mohl by se v prohlížečce zobrazovat údaj o tom, o jaké jde číslo? V seznamu článků v Archivu je tento údaj uveden.

**Comment by albertoh** [comment]
Pokud myslite napis nad seznamu clanku, ja ten udaj tam vidim.

**Comment by krizovj** [comment]
No, já myslím,přímo v seznamu u každého článku, jestli by to šlo. Je to jen nahoře nad tím seznamem, ale mohlo by to být i u každého článku, údaj o ročníku tam je taky. U Knih a dějin to tak úplně nevadí že to tam není, ptž mají jen jedno číslo za rok, ale jiné časopisy vycházejí třeba i jednou za měsíc a to už by tam ten údaj asi chtělo mít.

**Comment by albertoh** [comment]
Nechal jsem ten horni popisek porad videt. Myslim si, ze bude to prehlednej. Souhlas?

**Comment by krizovj** [comment]
super, díky.

**Comment by krizovj** [comment]
Myslím, že to souvisí s tím, že se tam ukotvil ten horší řádek. Když se třeba z archivu nebo z výsledků vyhledávání prokliknu na konkrétní článek, tak nevidím celý jeho název, jen autora. 
![bez nazvu](https://user-images.githubusercontent.com/21309201/43128665-d14e3cfc-8f32-11e8-84b1-05c3b6121c87.jpg)
Nevím, jestli je to z toho pochopitelné.
